src/core/messaging.cpp

```diff
@@ -28,13 +28,13 @@ void messaging_init() {
 }
 
 
-void send_message(const json &msg) {
+void send_message(const string type, const json &msg) {
```

Comment:
How is this better? Having this seems to just add more redundant code to node/socket.js